src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregationManager.cs

```diff
@@ -97,6 +97,7 @@ internal sealed class AggregationManager
             _listener.SetMeasurementEventCallback<short>((i, m, l, c) => ((InstrumentState)c!).Update((double)m, l));
             _listener.SetMeasurementEventCallback<byte>((i, m, l, c) => ((InstrumentState)c!).Update((double)m, l));
             _listener.SetMeasurementEventCallback<decimal>((i, m, l, c) => ((InstrumentState)c!).Update((double)m, l));
+            _listener.SetMeasurementEventCallback<uint>((i, m, l, c) => ((InstrumentState)c!).Update((uint)m, l));
```

Comment:
This looks odd; all the other types are cast to double.

Reply:
@KalleOlaviNiemitalo I changed it as you say.